chat_with_nerf/visual_grounder/picture_taker.py

```diff
@@ -128,12 +128,21 @@ def take_picture(self, query: str, session_id: str) -> tuple[list[ImageRef], str
         top_indices = np.argpartition(flattened_values, -percentage_points)[
             -percentage_points:
         ]
+        logger.info(
+            "Export RGB GLB files highlighted the top 5% values with color of red..."
+        )
```

Comment:
move this logging to line 216. also the "5 percent" and "red" is no longer true: change to "0.5%" and "green"